controllers/provisioning_controller.go

```diff
@@ -183,18 +178,6 @@ func (r *ProvisioningReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error
 		return ctrl.Result{}, err
 	}
 
-	if err := provisioning.ValidateBaremetalProvisioningConfig(baremetalConfig); err != nil {
```

Comment:
This is one thing that I am not totally sure about. Can the CR get created/updated when the operator is not running?
I might leave this in place just out of precaution.

Reply:
I don't see any harm in leaving it in.